src/common/gui/builtinThemes.ts

```diff
@@ -52,9 +54,10 @@ export const stateBgFocus = "rgba(139,139,139,0.29)"
 export const stateBgActive = "rgba(139,139,139,0.38)"
 
 type Themes = Record<ThemeId, Theme>
-export const themes: Themes = {
-	light: Object.freeze({
-		themeId: "light",
+
+export const themes = (): Themes => {
+	const lightRed = Object.freeze({
+		themeId: !client.isCalendarApp() ? "light" : "light_secondary",
```

Comment:
I think having theme id depend on the app makes it more confusing. I would make four theme IDs:
`light_red`, `dark_red`, `light_blue`, `dark_blue` and then just map them to the appropriate names in UI.

This makes it a bit harder to migrate but if we need to migrate anyway it doesn't make a big difference.'

Doing this will allow to make the themes an object. If you prefer to make it a function for flexibility then I would recommend to at least memoize it.

Reply:
I went with this approach because we need to switch between light and dark and this would lead to checking which app we're using anyway and changing things on android, ios and desktop to switch between light_red/dark_red or light_bue/dark_blue depending on the app.

Maybe just memorising will be enough, since if we have to migrate it won't make a big difference?

Reply:
Ideally the apps should parse `auto:light|dark` into the themes they should be using, it just wasn't implemented.
I won't insist on this, I think it would be cleaner but I'm okay with your proposal as well.